README.md

```diff
@@ -87,6 +87,26 @@ And use it to verify the signature:
 cosign verify --key strimzi.pub quay.io/strimzi/operator:latest --insecure-ignore-tlog=true
 ```
 
+## Software Bill of Materials (SBOM)
+
+From the 0.38.0 release, Strimzi publishes the software bill of materials (SBOM) of our containers.
+The SBOMs are published as an archive with `SPDX-JSON` and `Syft-Table` formats signed using cosign.
```

Comment:
any reasons for choosing SPDX format? I am new to Syft but I see it can generate other formats as well (i.e. CycloneDX).

Reply:
SPDX is the Linux Foundation format. So I picked it as it is closer to us as a project (CNCF). No other reasons from my side to be honest.

Reply:
Do you see any reasons why to use CycloneDX instead of SPDX?

Reply:
No tbh and admit my ignorance I was totally new to these formats so I slightly read about them. I was curious about your choice, but I think that it makes sense.

Reply:
I will be honest with you. The only reason I made my decision was on the LF / CNCF relationship as I did not have any other preferences one way or the other 🤷. 

The archive also has the Syft-Table format that you can easily read in a text editor as a human. But I had not much else to decide on for the machine-readable formats.